js/test/ValidateSPV.test.js

```diff
@@ -159,4 +171,89 @@ describe('ValidateSPV', () => {
       }
     });
   });
+
+  describe('#validateHeader', () => {
+    it('returns true if all elements of the bitcoin header are valid', () => {
```

Comment:
`returns true if the header object is syntactically valid`

Reply:
Fixed here as well as in function documentation